erpnext/controllers/accounts_controller.py

```
@@ -2269,9 +2269,9 @@
 		)
 
 		if not paid_amount:
-			if self.doctype in frappe.get_hooks("advance_payment_receivable_doctypes"):
+			if self.doctype in self.get_advance_payment_doctypes(type="receivable"):
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'type' in method call

(E1123)